movies/movies

```diff
@@ -158,28 +166,16 @@ getConfiguredPython || exit 1
 getConfiguredClient || exit 1
 checkInternet || exit 1 # check if we have a valid internet connection if this isnt true the rest of the script will not work so stop here
 
-while getopts "uvh" opt; do
-  case $opt in
-    \?)
-      echo "Invalid option: -$OPTARG" >&2
-      exit 1
-    ;;
-    h)
-      usage
-      exit 0
-    ;;
-    v)
-      echo "Version $currentVersion"
-      exit 0
-    ;;
-    u)
-      update
-      exit 0
-    ;;
-    :)
-      echo "Option -$OPTARG requires an argument." >&2
-      exit 1
-    ;;
+while getopts 'ud:hv' flag; do
```

Comment:
Im confused what the colon here after the d flag implies. I use it to say this option requires and argument but that is not the case here since you removed that part from the options parser (which doesn't matter since none of the options required arguments anyways.